### PR TITLE
Submitting NTop version 0.3.20

### PR DESCRIPTION
**Pull request by gsass1**
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ x Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/185978)

### CODE REVIEW

**Comment by gsass1** [comment]
@microsoft-github-policy-service agree

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> Executable C:\Users\validator\AppData\Local\Microsoft\WinGet\Packages\gsass1.NTop__DefaultSource\ntop.exe returned exit code: -1073741515
> Executable C:\Users\validator\AppData\Local\Microsoft\WinGet\Links\ntop.exe returned exit code: -1073741515

Automated error analysis suggests -1073741515 may mean STATUS_DLL_NOT_FOUND - possibly due to a dependency on `Microsoft.VCRedist.2015+.x64` (or `.x86`).

(Automated response - build 897.)

File: manifests/g/gsass1/NTop/0.3.20/gsass1.NTop.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=NVS1.5-1-19041-1682
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: gsass1.NTop
+PackageVersion: 0.3.20
+MinimumOSVersion: 10.0.0.0
+InstallerType: portable
+Commands:
+- ntop
+Installers:
```
Comment (stephengillie) [inline manifests/g/gsass1/NTop/0.3.20/gsass1.NTop.installer.yaml:10]:
```suggestion
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
Installers:
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
Hi @gsass1,

This package appears to have a dependency on `Microsoft.VCRedist.2015+.x64`:
image

**Review by stephengillie — APPROVED** [review]

**Pull request merged**